templates/certificates/dynamic-styles.php

```diff
@@ -27,11 +27,11 @@
 <?php foreach ( $fonts as $font ) : ?>
 	<?php if ( ! empty( $font['href'] ) ) : ?>
 		<?php
-		if ( ! $gfonts_preconnet && false !== strpos( $font['href'], 'fonts.googleapis.com' ) ) :
+		if ( ! $gfonts_preconnet && false !== strpos( $font['href'], LLMS_PLUGIN_URL . '/assets/css/localstyles.css'  ) ) :
 			$gfonts_preconnet = true;
 			?>
-			<link rel="preconnect" href="https://fonts.googleapis.com">
-			<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+			<link rel="preconnect" href="<?LLMS_PLUGIN_URL ?>">
+			<link rel="preconnect" href="<?LLMS_PLUGIN_URL ?>" crossorigin>
```

Comment:
This is problematic still.

First, we don't need to preconnect to the site as we're already connected and second this is designed to preconnect to google when google fonts are being used so the change actually should be undone.

Also, if you got the `<link>` elements to output they'd have an empty `href` since the constant isn't being echoed.

Finally, short open tags are generally frowned upon in WordPress since not all server environment support them.